configs/flags.yml

```diff
@@ -1,3 +1,5 @@
 flags:
   - name: performance.backgroundJsonSchemaValidation
     enabled: true
+  - name: heartbeat.fail_sync
```

Comment:
The docs are up yet, but the existing feature-flags are all camelCased.  Could we camelCase this one for consistency?

Reply:
Done